src/drivers/vfs.rs

```diff
@@ -0,0 +1,49 @@
+use std::collections::{hash_map::HashMap, hash_set::HashSet};
+use std::io;
+use std::path::Path;
+
+use cargo_toml::AbstractFilesystem;
+
+use crate::helpers::PathExt;
+
+#[derive(Debug)]
+pub(super) struct Vfs(HashMap<Box<Path>, HashSet<Box<str>>>);
```

Comment:
Can we have a comment with a little description of what this type is representing/storing?

Reply:
Yes!

Reply:
Done!